integration-tests/src/test/java/org/apache/fineract/integrationtests/ExternalIdSupportIntegrationTest.java

```diff
@@ -900,6 +900,8 @@ public void loan() {
             assertEquals(actualDate, loanApprovalResult.getApprovalDate());
             assertEquals(1000.0, loanApprovalResult.getApprovalAmount());
             assertEquals(1000.0, loanApprovalResult.getNetDisbursalAmount());
+            assertNotNull(loanApprovalResult.getCurrency());
```

Comment:
Would you mind to rewrite to check the value of the currency code from the loan approval response is matching with the loan product currency?

Also you have added the currency for disbursal template as well. Please write test for that one as well!

Reply:
Done! PR updated with the cnanges requested